docs/contributing/README.md

```diff
@@ -136,12 +136,12 @@ The users below are permanent on the test servers, and will not be deleted. Howe
 
 Name | Username | Login email | Login Password
 :---- | :---- | :---- | :----
-Application Administrator | applicationAdminUser | applicationadministrator@pilcrow.dev | adminPassword!@#
-Publication Administrator | publicationAdministrator | publicationAdministrator@pilcrow.dev | publicationadminPassword!@#
-Publication Editor | publicationEditor | publicationEditor@pilcrow.dev | editorPassword!@#
-Review Coordinator for Submission | reviewCoordinator | reviewCoordinator@pilcrow.dev | coordinatorPassword!@#
-Reviewer for Submission | reviewer | reviewer@pilcrow.dev | reviewerPassword!@#
-Regular User | regularUser | regularuser@pilcrow.dev | regularPassword!@#
+Application Administrator | applicationAdminUser | `applicationadministrator@pilcrow.dev` | adminPassword!@#
+Publication Administrator | publicationAdministrator | `publicationAdministrator@pilcrow.dev` | publicationadminPassword!@#
+Publication Editor | publicationEditor | `publicationEditor@pilcrow.dev` | editorPassword!@#
+Review Coordinator for Submission | reviewCoordinator | `reviewCoordinator@pilcrow.dev` | coordinatorPassword!@#
+Reviewer for Submission | reviewer | `reviewer@pilcrow.dev` | reviewerPassword!@#
+Regular User | regularUser | `regularuser@pilcrow.dev` | regularPassword!@#
```

Comment:
```suggestion
Application Administrator | applicationAdminUser | `applicationadministrator@pilcrow.dev` | `adminPassword!@#`
Publication Administrator | publicationAdministrator | `publicationAdministrator@pilcrow.dev` | `publicationadminPassword!@#`
Publication Editor | publicationEditor | `publicationEditor@pilcrow.dev` | `editorPassword!@#`
Review Coordinator for Submission | reviewCoordinator | `reviewCoordinator@pilcrow.dev` | `coordinatorPassword!@#`
Reviewer for Submission | reviewer | `reviewer@pilcrow.dev` | `reviewerPassword!@#`
Regular User | regularUser | `regularuser@pilcrow.dev` | `regularPassword!@#`
```

I like you choice of backticks here. I believe the passwords should be given the same treatment.